src/app.ts

```diff
@@ -1245,6 +1245,12 @@ function registerIoSignAPIRoutes(
     bearerSessionTokenAuth,
     toExpressHandler(ioSignController.getQtspClausesMetadata, ioSignController)
   );
+
```

Comment:
Isn't this supposed to be added to https://github.com/pagopa/io-backend/blob/master/api_io_sign.yaml as well?

Reply:
Yes, otherwise an error will also be generated during start!